### ISSUE TITLE
[Bug]: oxford_gov_uk just reports a calendar as off and no dates for collections

### ISSUE THREAD

**Issue by matt05d** [mampfes/hacs_waste_collection_schedule#1761]
### I Have A Problem With:

A specific source

### What's Your Problem

Hello and thank you for this integration and any help in advance!
The Oxford source no longer works.
I have tried my own details and the example in the documentation and both return the same result.

### Source (if relevant)

oxford_gov_uk

### Logs

```Shell
Error in the logs is:

Logger: waste_collection_schedule.source_shell
Source: custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py:136
Integration: waste_collection_schedule ([documentation](https://github.com/mampfes/hacs_waste_collection_schedule#readme))
First occurred: 09:57:29 (1 occurrences)
Last logged: 09:57:29

fetch failed for source Oxford City Council: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 134, in fetch entries = self._source.fetch() ^^^^^^^^^^^^^^^^^^^^ File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/oxford_gov_uk.py", line 55, in fetch for paragraph in collection_soup.find("div", class_="editor").find_all("p"): ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ AttributeError: 'NoneType' object has no attribute 'find_all'
```

### Relevant Configuration

```YAML
Source config:

waste_collection_schedule:
    sources:
    - name: oxford_gov_uk
      args:
        uprn: 10012080****
        postcode: OX1****

Sensors config:

- platform: waste_collection_schedule
  name: next_recycling_collection
  details_format: generic
  types:
    - Recycling
  value_template: '{% if value.daysTo == 0 %}Today{% elif value.daysTo == 1 %}Tomorrow{% else %}in {{value.daysTo}} days{% endif %}'

- platform: waste_collection_schedule
  name: next_food_collection
  details_format: generic
  types:
    - Food
  value_template: '{% if value.daysTo == 0 %}Today{% elif value.daysTo == 1 %}Tomorrow{% else %}in {{value.daysTo}} days{% endif %}'
- platform: waste_collection_schedule
  name: next_refuse_collection
  details_format: generic
  types:
    - Refuse
  value_template: '{% if value.daysTo == 0 %}Today{% elif value.daysTo == 1 %}Tomorrow{% else %}in {{value.daysTo}} days{% endif %}'
- platform: waste_collection_schedule
  name: next_garden_collection
  details_format: generic
  types:
    - Garden
  value_template: '{% if value.daysTo == 0 %}Today{% elif value.daysTo == 1 %}Tomorrow{% else %}in {{value.daysTo}} days{% endif %}'
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [ ] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by 5ila5** [comment]
Fixed by #1766
This will be part of release 1.46.0 or is available now as master (git) version (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)

**Comment by matt05d** [comment]
Thank you for the quick fix
